nntrainer/tensor/tensor_v2.h

```diff
@@ -504,18 +700,74 @@ class TensorV2 {
     return ((T *)getData())[idx];
   }
 
+  /**
+   * @brief Get the Value thinking that it is padded
+   * for example, for the tensor (virtually padded) below,
+   * getValue(0, 0, 2, 2, 1, 1, .0f) will return 5
+   * padding available for height and width axis for now
+   * 0 0 0 0 0
+   * 0 1 2 3 0
+   * 0 4 5 6 0
+   * 0 7 8 9 0
+   * 0 0 0 0 0
+   * @param b batch index
+   * @param c channel index
+   * @param h height index
+   * @param w width index
+   * @param ph padding height
+   * @param pw padding width
+   * @return float value
+   */
+  template <typename T = float>
+  const T getValuePaddedVirtual(unsigned int b, unsigned int c, unsigned int h,
```

Comment:
is there any use cases ?

Reply:
no use case other than a unit test. we could remove this if it's not a used feature anymore